packages/styles/src/components/form-check.scss

```diff
@@ -83,9 +83,10 @@
       border-radius: 50%;
 
       &:checked::after {
-        border: spacing.$size-micro solid color.$white;
```

Comment:
The border addition was erroneously introduced in https://github.com/swisspost/design-system/pull/2086